psi_cardinality/go/server/server_test.go

```diff
@@ -3,6 +3,7 @@ package server
 import (
 	"github.com/openmined/psi-cardinality/client"
 	"testing"
+	"bytes"
```

Comment:
Please run gofmt(and golint) over the modified files.

I will add the checks to the workflows too.

Reply:
Done.